kitsune/search/v2/documents.py

```diff
@@ -162,6 +162,14 @@ def get_queryset(cls):
             .annotate(es_question_num_votes=Count("votes"))
         )
 
+    @classmethod
+    def db_to_es_id(cls, db_id):
+        return "q_{}".format(db_id)
+
+    @classmethod
+    def es_to_db_id(cls, es_id):
+        return es_id[2:]
+
 
 class AnswerDocument(QuestionDocument):
```

Comment:
Can't we just apply some logic in a `prepare` method for Answers? I assume that the problem arises by overwriting the ID with Answer objects. If this is the case we could just call the super method and apply some logic for the ID. We can even do this with Questions if needed.